src/legacy/ui/public/saved_objects/components/saved_object_finder.scss

```diff
@@ -0,0 +1,3 @@
+.euiFilterSelectItem--heading:hover {
```

Comment:
Please don't overwrite EUI (`.eui__`) classes directly. This will have a global affect. If you're just altering the current implementation, you should pass a custom `className` to the component usually prefixed by the current scope (in this case something like `.svoFilterSelectItem`). 

Unless you're finding that it's a bug in EUI, in which case it should be fixed in the rep, so you can make an issue for it there.